### PR TITLE
Add filter command

### PR DESCRIPTION
**Pull request by zzs-RedCocoon**
Add filter command; change toString in Movie.java

### CODE REVIEW

**Review by choongzhanhong — COMMENTED** [review]
LGTM. Minor style changes

File: src/main/java/Movie.java [hunk 1]
```diff
@@ -101,9 +101,16 @@ public String[] getGenres() {
         return genres;
     }
 
+    public String getGenresString(){
+        String genres = "";
+        for(String genre: getGenres()){
```
[the remark this answers is not in the document]
Comment (zzs-RedCocoon) [reply src/main/java/Movie.java]: Space added!

File: src/main/java/MovieList.java [hunk 2]
```diff
@@ -87,6 +88,19 @@ public void list() {
         }
     }
 
+    public void filter(String genre){
```
[the remark this answers is not in the document]
Comment (zzs-RedCocoon) [reply src/main/java/MovieList.java]: Added.

File: src/main/java/Movie.java [hunk 3]
```diff
@@ -101,9 +101,16 @@ public String[] getGenres() {
...
+            genres += genre;
+        }
+        return " [" + genres + "]";
+    }
     @Override
     public String toString() {
-        return String.format("%s (%d)", this.title, this.year);
+        return String.format("%s (%d)" + getGenresString(), this.title, this.year);
```
[the remark this answers is not in the document]
Comment (zzs-RedCocoon) [reply src/main/java/Movie.java]: Sorry, I have changed it.

**Pull request merged**